libs/core/src/stories/getting-started.docs.mdx

```diff
@@ -0,0 +1,96 @@
+import { Meta, Story } from '@storybook/blocks';
+
+<Meta title="Resources/Get Started"/>
+
+# Getting Started
+
+Pine is a library of Web Components crafted by the Design System Services team at Kajabi. It provides a robust foundation for building cohesive and accessible user interfaces. Designed with scalability and performance in mind, Pine enables developers and designers to collaborate seamlessly on consistent UI experiences.
+
+## Repository
+
+The source code can be found in the [Github Repo](https://github.com/Kajabi/pine).
+
+## Design Files
+
+The component designs can be found in the [Figma Project](https://www.figma.com/design/CC1YmaGKHnsvB28yLY9mEH/%E2%9D%96-Mercury-components?m=auto&node-id=0-1&t=xybx2d1s26U4p0Jd-1).
```

Comment:
This file is an internal Figma file. Should we be exposing this, since it won't work for the public?

Reply:
Good call. Will remove.